tests/manage/test_verify_all_fields_in_sc_yaml_with_oc_describe_sc.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+import logging
+
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.framework.testlib import ManageTest, tier1
+from tests.fixtures import create_ceph_block_pool
+
+log = logging.getLogger(__name__)
+RBD_SC_OBJ = None
+CEPHFS_SC_OBJ = None
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    This is a test fixture
+    """
+    def finalizer():
+        teardown()
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def setup():
+    """
+    Setting up the environment
+    *. Creating RBD and CephFS Secret
+    *. Creating Ceph block pool
+    """
+    global RBD_SECRET_OBJ
+    log.info("Creating RBD Secret")
+    RBD_SECRET_OBJ = helpers.create_secret(constants.CEPHBLOCKPOOL)
+
+    log.info("Creating CEPHFS Secret")
+    global CEPHFS_SECRET_OBJ
+    CEPHFS_SECRET_OBJ = helpers.create_secret(constants.CEPHFILESYSTEM)
+
+    log.info("Creating RBD Ceph pool")
+    global POOL
+    POOL = helpers.create_ceph_block_pool()
+
+
+def teardown():
+    """
+    Tearing down the environment
+    """
+    log.info(f"Deleting RBD storage class: {RBD_SC_OBJ.name}")
+    assert RBD_SC_OBJ.delete()
+    log.info(f"Deleting CEPHFS storage class: {CEPHFS_SC_OBJ.name}")
+    assert CEPHFS_SC_OBJ.delete()
+    log.info("RBD and CephFS Storage classes deleted successfully")
+    log.info(f"Deleting RBD Secret: {RBD_SECRET_OBJ}")
+    assert RBD_SECRET_OBJ.delete()
+    log.info(f"Deleting CEPHFS Secret: {CEPHFS_SECRET_OBJ}")
+    assert CEPHFS_SECRET_OBJ.delete()
+    log.info("RBD and CephFS Secret's deleted successfully")
+    log.info(f"Deleting Ceph block pool {POOL}")
+    assert POOL.delete()
+    log.info(f"Ceph block pool:{POOL} deleted successfully")
+
+
+@tier1
+@pytest.mark.usefixtures(test_fixture.__name__)
+@pytest.mark.usefixtures(create_ceph_block_pool.__name__)
+@pytest.mark.polarion_id("OCS-521")
```

Comment:
How this should be reported to Polarion, when two tests have the same Polarion ID? I think, that `polarion_id` marker should be used only on test level.

Reply:
In Polarion I wrote it as a single generic test case which covers both RBD and CephFS as this scenario needs to be tested on both RBD and CephFS.
May be given that it is a tier-1 case, we can fail the test case in polarion if atleast one fails.
Thoughts?
Let me know if you have any other suggestions.

Reply:
I will create two separate cases in polarion for each interface.

Reply:
Done, Created two separate test cases in polarion